packages/project-access/test/project/minUI5Version.test.ts

```diff
@@ -9,6 +9,7 @@ describe('getters for minUI5Version', () => {
     describe('getMinUI5VersionFromManifest', () => {
         const testVersions: Array<TestCase> = [
             { minUI5Version: undefined, expectedResult: undefined },
+            { minUI5Version: '${sap.ui5.dist.veresion}', expectedResult: '${sap.ui5.dist.veresion}' },
```

Comment:
```suggestion
            { minUI5Version: '${sap.ui5.dist.version}', expectedResult: '${sap.ui5.dist.version}' },
```
Typo? Same below.

Reply:
Thanks for the review @devinea . Typo corrected with next commit